quiz.js

```diff
@@ -60,10 +63,15 @@ const questions = [
     
   }
 
+  // verify currentQuestion. if it's correct increase correctAnswers counter
+  function verify() {
```

Comment:
sposób na odnalezienie currentAnswer bardzo by się tu przydał. Wprawdzie nie mamy teraz jeszcze wyświetlania żadnych odpowiedzi, jest to trochę kłopotliwe ale można pomyśleć na razie teoretycznie. Odpowiedzi pojawią się w formie identycznej jak w naszym pliku html, czyli inputy (typu radio). Musimy wybrać (np selektorem) ten input, który jest zaznaczony a potem odczytać jego wartość (value) bo tam przechowywana jest informacja czy jest to button a,b, czy c). Czyli największy problem to będzie napisać odpowiedni query selektor. Podpowiedź: pseudoklasa "checked"

Reply:
ok, dzięki za podpowiedzi :)